### PR TITLE
Overmind over Hivemind to allow developer to run byebug in Rails

### PR DESCRIPTION
**Pull request by user512**
To allow developer to run debugger in convene-web.

### CODE REVIEW

**Comment by user512** [comment]
Blocked by https://github.com/zinc-collective/convene/pull/225, rebase `development` after #225 is merged in.

File: .github/workflows/test-convene-web.yml [hunk 1]
```diff
@@ -34,12 +34,12 @@ jobs:
...
-    - name: Install Hivemind
+    - name: Install Overmind
       run: |
         sudo mkdir ~/bin/
-        sudo wget -O ~/bin/hivemind.gz https://github.com/DarthSim/hivemind/releases/download/v1.0.6/hivemind-v1.0.6-linux-386.gz
-        sudo gunzip ~/bin/hivemind.gz
-        sudo chmod +x ~/bin/hivemind
+        sudo wget -O ~/bin/overmind.gz https://github.com/DarthSim/overmind/releases/download/v2.2.2/overmind-v2.2.2-linux-386.gz
+        sudo gunzip ~/bin/overmind.gz
+        sudo chmod +x ~/bin/overmind
```
Comment (zspencer) [inline .github/workflows/test-convene-web.yml:42]: Do we need this since `brew install overmind` works in CI?

**Review by zspencer — APPROVED** [review]
Thanks Tom!

File: .github/workflows/test-convene-web.yml (hunk 1 above)
Comment (user512) [reply .github/workflows/test-convene-web.yml:42]: Right! Let me get rid of this.

**Pull request merged**